src/cmd/job/attach.c

```diff
@@ -1284,6 +1284,21 @@ static void initialize_attach_statusline (struct attach_ctx *ctx,
     }
 }
 
+static void setup_signal_handlers (struct attach_ctx *ctx, flux_reactor_t *r)
+{
+    ctx->sigint_w = flux_signal_watcher_create (r,
+                                                SIGINT,
+                                                attach_signal_cb,
+                                                &ctx);
+    ctx->sigtstp_w = flux_signal_watcher_create (r,
+                                                 SIGTSTP,
+                                                 attach_signal_cb,
+                                                 &ctx);
```

Comment:
Should be `ctx` here and not `&ctx` I think?

Reply:
D'oh. I'm not sure how this got in here, it would cause a segfault in the tests. This could be a rebase error. Anyway, will fix.

Reply:
Good catch btw! Thanks!

Reply:
I think it is just a rebase error, because it's fixed in a later commit.

Reply:
Thanks. I'll fix that.

Reply:
Done! Will set MWP. Thanks.